app/assets/javascripts/incident_reports/edit.coffee

```diff
@@ -39,8 +39,9 @@ addInjuredPassengerFields = (event) ->
 
 deleteFields = (fieldsSelector) ->
   fields = $(fieldsSelector).last()
-  fields.find('input').remove()
-  fields.remove()
+  fields.find('input').val('').prop 'checked', false
```

Comment:
Not sure if this is ideal. I mean, the delete box is still visible.

Reply:
A new problem I've run into is that if a user clicks "delete" on the last passenger, then they have to uncheck the `Were passengers injured?` checkbox or else the report doesn't save (says the injured passenger fields have to be filled out.)

Reply:
I could do more coffeescript. But I don't know if I want to go down that rabbit hole.

Reply:
No, I think that's correct. If passengers were injured, their information is required. That's a business requirement, therefore a model validation, therefore correct form behavior.